README.md

```diff
@@ -276,6 +304,10 @@ Compose.prototype.map = function(f) {
 
 #### `sequence` method
 
+```hs
+sequence :: Apply f, Traversable t => t (f a) ~> (b -> f b) -> f (t a)
```

Comment:
In practice yes, but I decided to just use the `Apply` constraint here because the `of` function (or any function that can lift a value into the `Apply`) has to be provided by the caller, so the constraint is then just that the type returned by the given function must be an `Apply`.